tests/test_integrated_channels/test_cornerstone/test_exporters/test_content_metadata.py

```diff
@@ -327,7 +327,7 @@ def test_transform_estimated_hours(self, item_course_runs, expected_duration):
                         "enrollment_start": None,
                         "pacing_type": "instructor_paced",
                         "end": "2013-02-05T05:00:00Z",
-                        "start": "2023-02-05T05:00:00Z",
+                        "start": "2013-02-05T05:00:00Z",
```

Comment:
How was this working??? My changes shouldn't have caused this to fail, this should have just failed on its own... I'm not sure how this got into master 😕

Reply:
ahhh it's fine on master... so this is most likely a rebasing issue.

Reply:
tl:dr **today** is the day that these hard coded dates switch to failing. In the words of @taliaedX ```The lesson we reiterated here was: Please don’t mix static and calculated dates. :slightly_smiling_face:```